tests/features/tmgmt/fast-track.feature

```diff
@@ -148,6 +148,7 @@ Feature: Fast track
     And I press "Apply"
     Then I should see "Revision state: Validated"
     And Poetry service received request should contain the following text:
+      | <produit>EDT</produit>                                        |
```

Comment:
The test of the "produit" value should be done after transition from "draft to "Needs review" above after the line 146. Here I propose to test on `<produit>TRA</produit>`